cmd/cluster-agent/api/v1/install.go

```diff
@@ -0,0 +1,152 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/DataDog/dd-go/log"
+	"github.com/gorilla/mux"
+)
+
+func Install(r *mux.Router) {
+	r.HandleFunc("metadata/nodes/{nodeName}/namespaces/{ns}/pods/{podName}", getPodMetadata).Methods("GET")
```

Comment:
Since there is now 3 path parameters it seemed justified to add additional context preceding each parameter. Let me know if you agree.

Reply:
Just a curiosity nit: As we only give one `node` and one `pod`, would we want to use singular in the url ?

Reply:
@devonboyer why do you think it's relevant to add these?
I think it's fine like that, but just to add other examples on something similar:
If we have a look to the current design of the logging endpoint, in Kubernetes, it's
`/containerLogs/${namespace}/${podName}/${containerName}`

Reply:
Alright, fair enough. I'll remove it.